examples/car/booking-create-request.json

```diff
@@ -0,0 +1,66 @@
+{
+  "leg": {
+    "to": {
+      "lat": 60.1693,
+      "lon": 24.9364,
+      "name": "EuroPark P-Simonkenttä",
+      "address": "streetName:Simonkatu|streetNumber:5|zipCode:00100|city:Helsinki"
+    },
+    "from": {
+      "lat": 60.1693,
+      "lon": 24.9364,
+      "name": "EuroPark P-Simonkenttä",
+      "address": "streetName:Simonkatu|streetNumber:5|zipCode:00100|city:Helsinki"
+    },
+    "mode": "SHARED_CAR",
+    "endTime": 1576607463612,
+    "agencyId": "CARSHARINGABC",
```

Comment:
AgencyId names were in usually in lower-dash-case...

Reply:
Fixed